examples/example_server.py

```diff
@@ -80,11 +124,25 @@ async def request_counter(self, ctx) -> None:
         self.sensors["counter-queries"].value += 1
 
     async def _service_task(self) -> None:
-        """Example service task that just broadcasts to clients."""
+        """Example service task that broadcasts to clients."""
         while True:
             await asyncio.sleep(10)
             self.mass_inform("hello", "Hi I am a service task")
 
+    async def _alter_sensors(self) -> None:
+        """Example service task that adds and removes a fixed sensor.
+
+        This demonstrate's the aggregate sensor's ability to add and remove
+        values from its total."""
+        await asyncio.sleep(10)
+        self.mass_inform("add", "I'm going to add a fixed sensor")
```

Comment:
katcp actually has a standard inform to indicate that sensors are changing.

Reply:
Is that `#interface-changed`? I just wanted to make explicit what was going to happen, for the benefit of the reader of the code.

Reply:
Yes, `#interface-changed`. I figure we might as well use the standard inform so that this is also an example of the right way to do dynamic sensors. I forget the exact syntax; I think there are ways to indicate an incremental change in the inform.